src/main/java/seedu/mark/logic/commands/results/CommandResult.java

```diff
@@ -30,6 +32,14 @@ public boolean isExit() {
         return false;
     }
 
+    public boolean isGoto() {
+        return false;
```

Comment:
```suggestion
        return getUrl() != null;
```

Because really it all depends on the url, then can remove the override in GotoCommandResult too.